src/test/librbd/fsx.cc

```diff
@@ -2393,12 +2393,16 @@ test(void)
 			log4(OP_SKIPPED, OP_WRITESAME, offset, size);
 			goto out;
 		}
+		break;
         case OP_COMPARE_AND_WRITE:
                 /* compare_and_write not implemented */
                 if (!ops->compare_and_write) {
                         log4(OP_SKIPPED, OP_COMPARE_AND_WRITE, offset, size);
                         goto out;
                 }
+		break;
+	default:
```

Comment:
Nit: I would leave off the ```default``` case so that the compilation will fail if a new enum is added and the switch statement isn't updated.